client/src/containers/user-dropdown/component.tsx

```diff
@@ -23,6 +24,8 @@ const UserDropdown: React.FC = () => {
 
   const handleSignOut = useCallback(() => signOut({ callbackUrl: '/auth/signin' }), []);
 
+  const userName = useMemo(() => getUserFullName(user), [user]);
```

Comment:
No need of using `useMemo` here, the operation is not that expensive to optimize it.